.github/workflows/ci.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       - name: "Enable yarn v4"
         run: |
           corepack enable
-          yarn set version 4.1.1
+          yarn set version berry
```

Comment:
Er det ikkje bedre å sette 4.5.1 her?

Reply:
Vet ikke. Vi setter berry i den andre workflowen. @jnatten har kanskje noen formeninger her, i og med at det var han som gjorde det først?

Reply:
Det kan hende jeg hadde formeninger en gang i tiden, men nå er det dårlig med det.
Generelt sett så liker jeg å låse versjoner for å minimere workflows og ting som brekker av seg selv.

Reply:
Om eg husker rett så betyr berry siste versjon, og når det kjem ny vil dette kunne føre til at yarn.lock endres i action, og dermed feiler det fordi det ikkje skal skje.